vpdq/python/vpdq.pyx

```diff
@@ -42,10 +42,10 @@ cdef extern from "vpdq/cpp/hashing/filehasher.h" namespace "facebook::vpdq::hash
         vector[vpdqFeature]& pdqHashes,
         string ffmpeg_path,
         bool verbose,
-        int seconds_per_hash,
+        float seconds_per_hash,
```

Comment:
The cython will do an automatic transformation from python float to CPP double. Since there is no inbuild python double, I am just using float in python and let cython do the conversion.
<img width="773" alt="image" src="https://user-images.githubusercontent.com/44279163/178533645-be530533-4d01-41b4-91fa-fdd95c412e21.png">